src/hooks/useNews.js

```diff
@@ -4,23 +4,26 @@ import { GetNews } from "../services/ApiService";
 
 const useNews = props => {
   const [newsRoomItems, setNewsItems] = useState([]);
+  const [newsRoomLoadMoreEndPoint, setNewsRoomLoadMoreEndPoint] = useState([]);
```

Comment:
not blocking, but i suspect this hook could just be `loadMoreEndpoint`

Reply:
done